src/test/java/com/quizapp/enterprise/EnterpriseApplicationTests.java

```diff
@@ -107,19 +107,19 @@ void verifyCorrectAnswer() {
      */
     @Test
     void verifyIncorrectAnswer() {
-        String userGuess = "Flubber";
-        String correctAnswer = "Blubber";
-
+//        String userGuess = "Flubber";
+//        String correctAnswer = "Blubber";
+//
+//
+//        ArrayList<Guess> gec = new GameService().checkGuess(userGuess, correctAnswer);
+//        boolean correct = true;
+//        for(Guess guess : gec) {
+//            if(!guess.IsCorrectLetter) {
+//                correct = false;
+//                break;
+//            }
+//        }
```

Comment:
Avoid committing commented-out code to version control, as version control offers better approaches:
1) If you are curious what used to be there, diff commits to see history.
2) If it is an experimental feature, use a feature branch to keep it separate from master.